packages/auth/src/grant/service.ts

```diff
@@ -112,22 +140,26 @@ async function initiateGrant(
 
     return {
       interact: {
-        redirect: config.resourceServerDomain + `/interact/${grant.interactId}`,
+        redirect: config.identityServerDomain + `/interact/${grant.interactId}`,
```

Comment:
I wonder whether we should move the build of the request response to the route and just return `grant` here.